From 028157081c0428bac1269078dd7f3360e3810824 Mon Sep 17 00:00:00 2001
From: Olav Vitters <olav@vitters.nl>
Date: Thu, 22 Sep 2016 21:00:29 +0200
Subject: [PATCH] Fix string format build error

backends/meta-input-settings.c:1245:27: error: format '%lx' expects
argument of type 'long unsigned int', but argument 3 has type 'guint64
{aka long long unsigned int}' [-Werror=format=]
---
 src/backends/meta-input-settings.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/backends/meta-input-settings.c b/src/backends/meta-input-settings.c
index ae17f7baa..74cf30afd 100644
--- a/src/backends/meta-input-settings.c
+++ b/src/backends/meta-input-settings.c
@@ -1242,7 +1242,7 @@ lookup_tool_settings (ClutterInputDeviceTool *tool,
 
   tool_id = clutter_input_device_tool_get_id (tool);
   device_id = get_tablet_settings_id (device, lookup_mapping_info (device));
-  path = g_strdup_printf ("/org/gnome/settings-daemon/peripherals/wacom/%s/%lx/",
+  path = g_strdup_printf ("/org/gnome/settings-daemon/peripherals/wacom/%s/%" G_GUINT64_FORMAT "/",
                           device_id, tool_id);
   tool_settings = tool_settings_new (tool, path);
   g_object_set_qdata_full (G_OBJECT (tool), quark_tool_settings, tool_settings,
